modules/catalog/build.gradle

```diff
@@ -29,6 +29,7 @@ dependencies {
     implementation project(':ignite-metastorage-api')
     implementation project(':ignite-vault')
     implementation project(':ignite-schema')
+    implementation project(':ignite-distribution-zones')
```

Comment:
Why does the catalog module need to depend on the distribution zones module? Is it a temporary solution (until we stop using configurations to store schemas)?

Reply:
This is a temporary dependency so that we can convert the table/index/zone configuration in "one place".

When we leave the configuration, this dependence should disappear in theory.

Reply:
fix it